Traceback (most recent call last):
  File "/Users/silviags/opt/miniconda3/envs/PyQM/lib/python3.9/site-packages/jupyter_core/utils/__init__.py", line 154, in wrapped
    asyncio.get_running_loop()
RuntimeError: no running event loop

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/silviags/opt/miniconda3/envs/PyQM/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/Users/silviags/opt/miniconda3/envs/PyQM/lib/python3.9/site-packages/nbclient/client.py", line 1319, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/silviags/opt/miniconda3/envs/PyQM/lib/python3.9/site-packages/jupyter_core/utils/__init__.py", line 158, in wrapped
    return loop.run_until_complete(inner)
  File "/Users/silviags/opt/miniconda3/envs/PyQM/lib/python3.9/asyncio/base_events.py", line 647, in run_until_complete
    return future.result()
  File "/Users/silviags/opt/miniconda3/envs/PyQM/lib/python3.9/site-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "/Users/silviags/opt/miniconda3/envs/PyQM/lib/python3.9/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/silviags/opt/miniconda3/envs/PyQM/lib/python3.9/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
print(np.loadtxt("quibits.txt"))
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mFileNotFoundError[0m                         Traceback (most recent call last)
[0;32m/var/folders/9b/hnjyg5ld18v94zj022rv_zzm0000gn/T/ipykernel_63415/2729334545.py[0m in [0;36m<cell line: 1>[0;34m()[0m
[0;32m----> 1[0;31m [0mprint[0m[0;34m([0m[0mnp[0m[0;34m.[0m[0mloadtxt[0m[0;34m([0m[0;34m"quibits.txt"[0m[0;34m)[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m
[0;32m~/opt/miniconda3/envs/PyQM/lib/python3.9/site-packages/numpy/lib/_npyio_impl.py[0m in [0;36mloadtxt[0;34m(fname, dtype, comments, delimiter, converters, skiprows, usecols, unpack, ndmin, encoding, max_rows, quotechar, like)[0m
[1;32m   1379[0m         [0mdelimiter[0m [0;34m=[0m [0mdelimiter[0m[0;34m.[0m[0mdecode[0m[0;34m([0m[0;34m'latin1'[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m   1380[0m [0;34m[0m[0m
[0;32m-> 1381[0;31m     arr = _read(fname, dtype=dtype, comment=comment, delimiter=delimiter,
[0m[1;32m   1382[0m                 [0mconverters[0m[0;34m=[0m[0mconverters[0m[0;34m,[0m [0mskiplines[0m[0;34m=[0m[0mskiprows[0m[0;34m,[0m [0musecols[0m[0;34m=[0m[0musecols[0m[0;34m,[0m[0;34m[0m[0;34m[0m[0m
[1;32m   1383[0m                 [0munpack[0m[0;34m=[0m[0munpack[0m[0;34m,[0m [0mndmin[0m[0;34m=[0m[0mndmin[0m[0;34m,[0m [0mencoding[0m[0;34m=[0m[0mencoding[0m[0;34m,[0m[0;34m[0m[0;34m[0m[0m

[0;32m~/opt/miniconda3/envs/PyQM/lib/python3.9/site-packages/numpy/lib/_npyio_impl.py[0m in [0;36m_read[0;34m(fname, delimiter, comment, quote, imaginary_unit, usecols, skiplines, max_rows, converters, ndmin, unpack, dtype, encoding)[0m
[1;32m    995[0m             [0mfname[0m [0;34m=[0m [0mos[0m[0;34m.[0m[0mfspath[0m[0;34m([0m[0mfname[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m    996[0m         [0;32mif[0m [0misinstance[0m[0;34m([0m[0mfname[0m[0;34m,[0m [0mstr[0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 997[0;31m             [0mfh[0m [0;34m=[0m [0mnp[0m[0;34m.[0m[0mlib[0m[0;34m.[0m[0m_datasource[0m[0;34m.[0m[0mopen[0m[0;34m([0m[0mfname[0m[0;34m,[0m [0;34m'rt'[0m[0;34m,[0m [0mencoding[0m[0;34m=[0m[0mencoding[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    998[0m             [0;32mif[0m [0mencoding[0m [0;32mis[0m [0;32mNone[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m    999[0m                 [0mencoding[0m [0;34m=[0m [0mgetattr[0m[0;34m([0m[0mfh[0m[0;34m,[0m [0;34m'encoding'[0m[0;34m,[0m [0;34m'latin1'[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;32m~/opt/miniconda3/envs/PyQM/lib/python3.9/site-packages/numpy/lib/_datasource.py[0m in [0;36mopen[0;34m(path, mode, destpath, encoding, newline)[0m
[1;32m    190[0m [0;34m[0m[0m
[1;32m    191[0m     [0mds[0m [0;34m=[0m [0mDataSource[0m[0;34m([0m[0mdestpath[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 192[0;31m     [0;32mreturn[0m [0mds[0m[0;34m.[0m[0mopen[0m[0;34m([0m[0mpath[0m[0;34m,[0m [0mmode[0m[0;34m,[0m [0mencoding[0m[0;34m=[0m[0mencoding[0m[0;34m,[0m [0mnewline[0m[0;34m=[0m[0mnewline[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    193[0m [0;34m[0m[0m
[1;32m    194[0m [0;34m[0m[0m

[0;32m~/opt/miniconda3/envs/PyQM/lib/python3.9/site-packages/numpy/lib/_datasource.py[0m in [0;36mopen[0;34m(self, path, mode, encoding, newline)[0m
[1;32m    530[0m                                       encoding=encoding, newline=newline)
[1;32m    531[0m         [0;32melse[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 532[0;31m             [0;32mraise[0m [0mFileNotFoundError[0m[0;34m([0m[0;34mf"{path} not found."[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    533[0m [0;34m[0m[0m
[1;32m    534[0m [0;34m[0m[0m

[0;31mFileNotFoundError[0m: quibits.txt not found.

